frontend/app/src/components/asset-manager/managed/ManagedAssetTable.vue

```diff
@@ -136,15 +137,28 @@ function getAsset(item: SupportedAsset) {
 const { setMessage } = useMessageStore();
 const { fetchIgnoredAssets, ignoreAsset, isAssetIgnored, unignoreAsset } = useIgnoredAssetsStore();
 const { isAssetWhitelisted, unWhitelistAsset, whitelistAsset } = useWhitelistedAssetsStore();
+const { show } = useConfirmStore();
 
 const { markAssetsAsSpam, removeAssetFromSpamList } = useSpamAsset();
 
 const { getChain } = useSupportedChains();
 
-async function toggleIgnoreAsset(identifier: string) {
-  if (get(isAssetIgnored(identifier)))
+async function toggleIgnoreAsset(asset: SupportedAsset) {
+  const { identifier, name, symbol } = asset;
+  if (get(isAssetIgnored(identifier))) {
     await unignoreAsset(identifier);
-  else await ignoreAsset(identifier);
+  }
+  else {
+    show({
+      message: t('ignore.confirm.message', {
+        asset: symbol || name,
+      }),
+      title: t('ignore.confirm.title'),
+      type: 'warning',
+    }, async () => {
+      await ignoreAsset(identifier);
+    });
+  }
```

Comment:
Would it make sense to move this to the `useIgnoreAssetStore` and reuse it to the other component too?